packages/js/src/Modules/Verto/util/debug.ts

```diff
@@ -0,0 +1,58 @@
+import { v4 as uuid } from 'uuid';
+import { WebRTCStats } from '@peermetrics/webrtc-stats';
+import {
+    DebugReportDataMessage,
```

Comment:
@farhat-ha could you configure prettier in your editor? https://app.getguru.com/card/iBGAejbT/Prettier